src/KristofferStrube.Blazor.FileSystemAccess/Options/DirectoryPickerOptions.cs

```diff
@@ -44,6 +44,7 @@ internal override ExpandoObject Serializable()
 public abstract class BaseDirectoryPickerOptions
 {
     public string? Id { get; set; }
+    public string? Mode { get; set; }
```

Comment:
```suggestion

    /// <remarks>
    /// Default value is <see cref="FileSystemPermissionMode.Read"/>.
    /// </remarks>
    public FileSystemPermissionMode Mode { get; set; } = FileSystemPermissionMode.Read;
```
We could use the `FileSystemPermissionMode` enum type here instead of a string to make it easier for the library consumer to parse a valid value. If you could accept this change then we also need to update the `Serializable` method to no longer check for null and we would need to update the sample as well.

To be extra nice we could add an XML Doc remarks